### PR TITLE
CDPT-487 Adjust probes to prevent restarts

### PR DESCRIPTION
**Pull request by digitalali-moj**
## Description
Adjust probes to allow some leeway for sidekiq to respond before restarting pod.

## Self-review checklist

* [ ] (1) Quick stakeholder demo done OR
* [ ] (2) ...bug with before and after screenshots
* [ ] (3) Tests passing
* [ ] (4) Branch ready to be merged (not work in progress)
* [ ] (5) No superfluous changes in diff
* [ ] (6) No TODO's without new ticket numbers
* [ ] (7) PR Prefixed with ticket number e.g. `CT-7654 ...`

### Related JIRA tickets
https://dsdmoj.atlassian.net/browse/CDPT-487

### CODE REVIEW

**Review by vertism — APPROVED** [review]
LGTM

File: k8s-deploy/production/deployment_sidekiq.yaml [hunk 1]
```diff
@@ -21,6 +21,7 @@ spec:
     spec:
       containers:
         - name: parliamentary-questions-rails-jobs
+          image: 754256621582.dkr.ecr.eu-west-2.amazonaws.com/pq-team/parliamentary-questions:pq-main-e15d73ca
```
Comment (Polsonby) [inline k8s-deploy/production/deployment_sidekiq.yaml:24]: Just wondering, was this change intended?

**Review by Polsonby — APPROVED** [review]
LGTM :)

File: k8s-deploy/production/deployment_sidekiq.yaml (hunk 1 above)
Comment (digitalali-moj) [reply k8s-deploy/production/deployment_sidekiq.yaml:24]: nooo! will change it :)
